Expensify/App
# Close Keyboard on iOS/Android using Swipeable View

<If necessary, assign reviewers that know the area or changes well. Feel free to tag any additional reviewers you see fit.>

### Details
I created SwipeableView to wrap ReportActionCompose Component. 

Not sure if I should make SwipeableView cross-platform and separate ReportView to have SwipeableView wrap ReportActionCompose in iOS instead. Please let me know what you suggest!  

### Fixed Issues
Fixes GH_LINK #1870 

### Tests
1. Click on a chat with someone
2. Type something (toggle software keyboard on if you're on iOS Simulator) 
3.  Swipe down on the message window
4. Verify swiping down closes the keyboard


### Tested On


### Screenshots


## IOS
https://user-images.githubusercontent.com/19339044/112417793-65d09280-8d50-11eb-92ea-3b957f9735c5.mov
